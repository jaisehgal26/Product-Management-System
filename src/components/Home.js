import React, { Suspense } from "react";
import Carousel from "./Carousel";
import Footer from "./Footer";
import Navbar from "./Navbar";

const Service = React.lazy(() => import("./Service"));
const Review = React.lazy(() => import("./Review"));
const Facts = React.lazy(() => import("./Facts"));
const Banner = React.lazy(() => import("./Banner"));
export default function Home() {
  return (
    <>
      <Navbar />
      <Carousel />
      <Suspense
        fallback={
          <div className="center spinner-grow text-warning" role="status">
            <span className="sr-only">Loading...</span>
          </div>
        }
      >
        <Facts />
        <Service />
        <Banner />
        <Review />
      </Suspense>
      <Footer />
    </>
  );
}
