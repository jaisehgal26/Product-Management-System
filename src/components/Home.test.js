import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Home from "./Home";

test("should render correctly", () => {
  const wrapper = toJson(shallow(<Home />));
  expect(wrapper).toMatchSnapshot();
});
