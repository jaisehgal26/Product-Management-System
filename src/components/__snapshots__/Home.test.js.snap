// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Fragment>
  <Navbar />
  <Carousel />
  <Suspense
    fallback={
      <div
        className="center spinner-grow text-warning"
        role="status"
      >
        <span
          className="sr-only"
        >
          Loading...
        </span>
      </div>
    }
  >
    <lazy />
    <lazy />
    <lazy />
    <lazy />
  </Suspense>
  <Footer />
</Fragment>
`;
